docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+db:
+  image: postgres:9.5
+  env_file: .env
+  ports:
+    - '5432:5432'
+web:
+  build: .
+  env_file: .env
+  command: bash -c "python back_end.py db upgrade && python back_end.py runserver -h 0.0.0.0"
+  working_dir: /code
+  volumes:
+    - .:/code
+  env_file: .env
+  ports:
+    - '5000:5000'
```

Comment:
What I suggest it to add a string
`#!/usr/bin/env python`
at the beginning
and add *executable* flag to the file
 so you won't need to add "python" in the command